### ISSUE TITLE
Allow user to delete account from Wallet UI

### ISSUE THREAD

**Issue by mikedotexe** [nearprotocol/near-wallet#539]
## Story

A user who runs into problems updating their smart contract because of state issues. If the state has changed and the contract is already initialized, the user [gets and error and the only resolution](https://stackoverflow.com/questions/60767120/getting-cannot-deserialize-the-contract-state-when-calling-rust-init-function) is to delete the account and remake it in Wallet. Deleting an account has to happen in `near-shell`. It would be great if they can delete the account in Wallet. The current method feels clunky and hacky.

As a user, I want to be able to quickly "fix" my broken account and redeploy my contract without having to switch tools for this.

## Acceptance Criteria

A button/link exists to delete an account from NEAR, requiring the user to confirm their name before performing. Perhaps after this execution, the user is redirect to create a new account.

## Unhappy Path

User has to use `near-shell` then go to Wallet.

**Comment by kcole16** [comment]
I agree that there are valid reasons for users to want to delete their account, and support should be added to allow for this (without needing to use `near-shell`). 

Also agree that the contract redeployment flow is hacky/incomplete at the moment. 

Generally, I'd expect users in the contract redeployment case to already be working out of `near-shell`, and likely to be managing their contract-related account keys outside of the wallet (assuming developers `near login` with their funded, master account, and then use `near create_account` to generate contract-specific accounts).
